docs/source/metricsinfo.rst

```diff
@@ -0,0 +1,46 @@
+Metrics
+=======
+
+Visualizing the running ML system is important to measure success rate of the system. There will is a dashboard
```

Comment:
Typo here "There will is a dashboard" did you mean to say "There is a dashboard" ?

Reply:
Fixed